config/packages/prod/sulu_http_cache.yaml

```diff
@@ -1,8 +1,3 @@
-fos_http_cache:
-    proxy_client:
-        symfony:
-            use_kernel_dispatcher: true # disable this when varnish is used
```

Comment:
the prod container can't be built correctly, if varnish is used and this configuration stays here with `use_kernel_dispatcher: false`